azavea/echo-locator
# Initial styling

## Overview

Setup Sass. This lays a foundation for styling the rest of the app.

Style auth page. 
Style global header. 
Style search page.


### Demo

![image](https://user-images.githubusercontent.com/128699/53755005-7967e280-3e83-11e9-95de-a54b7f29970b.png)

![image](https://user-images.githubusercontent.com/128699/53754882-34dc4700-3e83-11e9-9f1f-ffdc2fa63201.png)

![image](https://user-images.githubusercontent.com/128699/53754986-710fa780-3e83-11e9-9beb-323dd56a4c8d.png)


### Notes

- Renamed `index.css` to `taui.css` for clarity.
- Added `focus-visible` polyfill.
- The fewer Amplify components, the easier it is to style the app. So I stripped out a bunch of Amplify components from the global header that I don't think we needed. Holler if I it turns out we needed those.


## Testing Instructions

 * Load auth screen. Confirm it looks per above.
 * Login. Confirm search screen looks per above.
 * Search for accounts. 
 * Confirm clicking an account list item selects the account and opens map view.
 * Confirm deleting an account deletes the account.
